packages/documentation-site/patternfly-docs/content/get-started/community/community.md

```diff
@@ -22,15 +22,9 @@ import CalendarIcon from '@patternfly/react-icons/dist/esm/icons/calendar-alt-ic
 <div className="ws-community-page pf-m-light-100">
   <p>At the core of PatternFly is our community of people—in other words, our Flyers. Together, we celebrate creativity and foster a sense of teamwork and unity.</p>
 
-  <Grid sm={12} md={6} hasGutter className="pf-v5-u-mt-3xl">
-    <GridItem className="pf-v5-u-pr-xl">
-      <img src={communityPageHeader} alt="Website collaboration" />
-    </GridItem>
-    <GridItem>
-      <Title size="3xl" className="ws-title" headingLevel="h2">Who we are</Title>
+  <Title size="3xl" className="ws-title" headingLevel="h2">Who we are</Title>
```

Comment:
The spacing is gone because the `<Grid>` wrapper that previously contained this title had the `pf-v5-u-mt-3xl` utility class which adds the top spacing.  With that `<Grid>` element & class removed, we'll want to add the `pf-v5-u-mt-3xl` class back in here to the Title to get the spacing back.
```suggestion
  <Title size="3xl" className="ws-title pf-v5-u-mt-3xl" headingLevel="h2">Who we are</Title>
```